app/posts/cross-programme/our-refreshed-vision.md

```diff
@@ -0,0 +1,108 @@
+
+Examples of adopting such an approach would mean contract managers can self-serve and are not reliant on the digital team to do things like adjusting funding caps for NPQ intakes, setting up new training cohorts and varying terms and service fees for each contract.
+
+We landed on the following principle: we should always design and build with self-sufficiency in mind. This means asking the question: can a team without developers use this feature indefinitely?
+
+## Our missions
+
+There was some confusion about how we prioritise work given our vision above alongside continuous pressure to deliver the policies and programmes as new scenarios emerge on the ground or policy changes come into force that we must respond to.
+
+Alongside delivering the policies and programmes, we have 3 missions that should guide the strategic transformation of the service. As a programme, we need to focus on tightly managing the former, so we have the space to do the latter. This will be achieved through regular conversations where teams and programme leads come together to discuss the roadmap and agree priorities.
+
+Based on the pain points surfaced in the CPD discovery and the barriers to achieving our vision, we came up with the following three missions for the digital programme:
+
+1. We eliminate the administrative burden involved in teachers taking up DfE-funded training.
+
+2. We achieve increasing efficiency and flexibility in managing the delivery of the programmes and adapting to policy changes.
+
+3. We provide accurate data and insights to evaluate the impact of the policy reforms and inform improvements to the policies, programmes and services.
+
+## Our strategy for delivery
+
+In order to achieve these missions and our wider vision, there are some fundamental things we need to get in place first. We therefore broke down our strategy into three phases for delivery:
+
+1. Our services serve their core purposes and are flexible enough to adapt to future changes and opportunities
+2. We have maximised value for users and increased efficiency through automation
+3. We have stabilised the services and consolidated down
+
+## Phase 1 -- Our services serve their core purposes and are flexible enough to adapt to future changes and opportunities
+
+Our services still fall short of delivering their core functions and are incredibly rigid, making it difficult to respond to new scenarios and changes. The lack of flexibility means that any future big changes to the delivery model will require a significant amount of work -- potentially building again from scratch rather than building from what we already have.
+
+Our key goals in this phase are to:
+
+- Decouple the NPQ service from the ECF service so they are separate services and databases to eliminate unnecessary dependencies and simplify them. This should eliminate a lot of the problems we have around data integrity.
```

Comment:
technically each bullet point should not start with a capital letter and not be more than 1 sentence (without a full stop...)